content/blog/2016-09-02-promcon-2016-its-a-wrap.md

```diff
@@ -0,0 +1,152 @@
+---
+title: PromCon 2016 - It's a wrap!
+created_at: 2016-09-02
+kind: article
+author_name: Julius Volz
+---
+
+## What happened
+
+Last week, eighty Prometheus users and developers from around the world came
+together for two days in Berlin for the first-ever conference about the
+Prometheus monitoring system: [PromCon 2016](https://promcon.io/). The goal of
+this conference was to exchange knowledge, best practices, and experience
+gained using Prometheus. We also wanted to grow the community and help people
+build professional connections around service monitoring. Here are some
+impressions from the first morning:
+
+<!-- more -->
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr">Just landed at <a href="https://twitter.com/hashtag/promcon?src=hash">#promcon</a>. This is gonna be an exciting conference. <a href="https://t.co/2hsFaS32IK">pic.twitter.com/2hsFaS32IK</a></p>&mdash; Till Backhaus (@backhaus) <a href="https://twitter.com/backhaus/status/768705298940956672">August 25, 2016</a></blockquote>
```

Comment:
Tried replacing all the `pic.twitter.com/...` references with local image copies, but after the Twitter JS loads, it's still all `img`s pointing to Twitter. Somewhere, deeply nested, it becomes something like:

```html
<img class="NaturalImage-image" data-srcset="https%3A%2F%2Fpbs.twimg.com%2Fmedia%2FCqr9YxJWgAAUWBK.jpg%3Alarge 960w,https%3A%2F%2Fpbs.twimg.com%2Fmedia%2FCqr9YxJWgAAUWBK.jpg 960w,https%3A%2F%2Fpbs.twimg.com%2Fmedia%2FCqr9YxJWgAAUWBK.jpg%3Asmall 680w" width="960" height="720" title="View image on Twitter" alt="View image on Twitter" src="https://pbs.twimg.com/media/Cqr9YxJWgAAUWBK.jpg:small">
```

So I don't know if that's trivially possible without doing all the Twitter magic ourselves...